src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -3,7 +3,7 @@ import { Course } from 'app/entities/course.model';
 import { ExerciseGroup } from 'app/entities/exercise-group.model';
 import { Exam } from 'app/entities/exam.model';
 import { artemis } from '../../support/ArtemisTesting';
-import { CypressAssessmentType, CypressExamBuilder, convertCourseAfterMultiPart } from '../../support/requests/CourseManagementRequests';
+import { CypressAssessmentType, CypressExamBuilder, convertCourseAfterMultiPart, CypressExerciseType } from '../../support/requests/CourseManagementRequests';
```

Comment:
Code issue

Reply:
Thanks for the hint! 
I just noticed, that we have two exercise type enums. With my latest commit I removed this one, so we only have one within the cypress tests.